base/dcp_common.go

```diff
@@ -160,23 +161,19 @@ func (c *DCPCommon) getMetaData(vbucketId uint16) (
 	return value, lastSeq, nil
 }
 
-// RollbackEx should be called by cbdatasource - Rollback required to maintain the interface.  In the event
-// it's called, logs warning and does a hard reset on metadata for the vbucket.  Returns error if metadata
-// persistence fails
-func (c *DCPCommon) rollback(vbucketId uint16, rollbackSeq uint64) error {
-	WarnfCtx(c.loggingCtx, "DCP Rollback request.  Expected RollbackEx call - resetting vbucket %d to 0.", vbucketId)
-	c.dbStatsExpvars.Add("dcp_rollback_count", 1)
-	c.updateSeq(vbucketId, 0, false)
-	err := c.setMetaData(vbucketId, nil, true)
-	return err
+func (c *DCPCommon) Rollback(partition string, rollbackSeq uint64) error {
```

Comment:
Generally I don't think we want to be implementing the cbgt interfaces (Dest, Reciever) in DCPCommon - the idea with DCPCommon is to define the common handling of these interfaces.  

It looks like we're actually still implementing Rollback in dcp_dest.go - was there other motivation for defining this here?

Reply:
This was just a mistake I left behind.